tests/test_advection.py

```diff
@@ -73,7 +73,6 @@ def test_advection_zonal(lon, lat, depth, mode):
     }
     dimensions = {"lon": lon, "lat": lat}
     fieldset2D = FieldSet.from_data(data2D, dimensions, mesh="spherical", transpose=True)
-    assert fieldset2D.U._creation_log == "from_data"
```

Comment:
Why not test for this creation_log here? Does it hamper development to perform this test?

Reply:
I moved it to `test_fieldset_from_data()`, thought it was better suited there